site/docs/v1/tech/apis/identity-providers/_samlv2-request-body.adoc

```diff
@@ -101,9 +121,15 @@ ifdef::samlv2_idp_initiated[]
 This field must be set to `SAMLv2IdPInitiated`.
 endif::[]
 
+[field]#identityProvider.uniqueIdClaim# [type]#[String]# [optional]#Optional#::
+The name of the unique `userId` claim (attribute in the `Assertion` element) in the SAML response that FusionAuth uses to uniquely link the user. If this is not set, the `emailClaim` will be used.
```

Comment:
I don't think we want to say this is a `userId` claim. It is the name of the claim that will tell FusionAuth the unique Id of the user.